src/test/resources/pages/instructorFeedbackContribQuestionAddSuccess.html

```diff
@@ -845,10 +845,11 @@ <h5 class="col-sm-2">
               </div>
               <div class="well panel panel-default panel-body mce-content-body" data-original-title="Please enter the description of the question." data-placement="top" data-toggle="tooltip" id="questiondescription-1" rows="5" spellcheck="false" tabindex="9" title="">
                 <p>
-                  <br data-mce-bogus="1">
+                  more details
```

Comment:
@wkurniawan07 , I've checked it once again. It seems that we missed this bug while implementing question descriptions. So, these changes seems intended for me